### PR TITLE
Dreiländer Hack blog article

### PR DESCRIPTION
**Pull request by celine-sncf**
blog article in french & english about OSRD at the Dreilander Hack 2023

### CODE REVIEW

**Review by axrolld — APPROVED** [review]
🚀

**Comment by multun** [comment]
Celine's article

File: content/blog/news/2023/dreilander-hack/index.en.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+The 150 participants were divided into 16 working groups on a wide variety of topics such as: 
+* Imagining a simple interface for ordering train paths (an important subject for the future of OSRD); 
+* Offering passenger information adapted to different disabilities; 
+* Optimising the prioritisation of trains at junctions according to the number of passengers on board and on the platform (Motis project); 
+* Automatically identify energy-consuming trains in storage; 
+* Calculate forecast maintenance dates from a historical database for each train component; 
+* Detect speed measurement and GPS positioning errors on track inspection trains; 
+* Optimise maintenance work to limit lost capacity; 
+* Facilitating passenger information with a GPT chatbot; 
+* Designing an easy-to-use display and controlticketing interface for night trains across Europe.
```
Comment (eckter) [inline content/blog/news/2023/dreilander-hack/index.en.md:22]: I assume there's a typo on `controlticketing`

File: content/blog/news/2023/dreilander-hack/index.en.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## 24 hours to work together on a topic of common interest
+The 150 participants were divided into 16 working groups on a wide variety of topics such as: 
+* Imagining a simple interface for ordering train paths (an important subject for the future of OSRD); 
+* Offering passenger information adapted to different disabilities; 
+* Optimising the prioritisation of trains at junctions according to the number of passengers on board and on the platform (Motis project); 
+* Automatically identify energy-consuming trains in storage; 
+* Calculate forecast maintenance dates from a historical database for each train component; 
+* Detect speed measurement and GPS positioning errors on track inspection trains; 
+* Optimise maintenance work to limit lost capacity;
```
Comment (eckter) [inline content/blog/news/2023/dreilander-hack/index.en.md:20]:
```suggestion
* Automatically identifying energy-consuming trains in storage; 
* Calculating forecast maintenance dates from a historical database for each train component; 
* Detecting speed measurement and GPS positioning errors on track inspection trains; 
* Optimizing maintenance work to limit lost capacity; 
```
For consistency with the other point (either that or the other way around)

**Review by multun — APPROVED** [review]

File: content/blog/news/2023/dreilander-hack/index.en.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+We were delighted to be invited to take part thanks to the [OpenRail Foundation](https://openrailfoundation.org/), and this year's theme being "Open Railways", it would have been hard to refuse!
+
+## 24 hours to work together on a topic of common interest
+
+The 150 participants were divided into 16 working groups on a wide variety of topics such as:
+
+* Imagining a simple interface for ordering train paths (an important subject for the future of OSRD);
+* Offering passenger information adapted to different disabilities;
+* Optimising the prioritisation of trains at junctions according to the number of passengers on board and on the platform (Motis project);
```
Comment (celine-sncf) [inline content/blog/news/2023/dreilander-hack/index.en.md:19]:
```suggestion
* Optimizing the prioritisation of trains at junctions according to the number of passengers on board and on the platform (Motis project);
```

**Review by celine-sncf — COMMENTED** [review]
last typos

**Review by multun — APPROVED** [review]

**Pull request merged**